src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -1262,17 +1276,15 @@ Content-Type: application/xml
 #### Response
 The response is the same as in Example 1.
 
+## <a name="postut"></a>[Update a Trip](/api-reference/travel/itinerary-tmc-thirdparty/index.html#postdetails)
```

Comment:
This looks like an old link from a previous site version, please remove

Reply:
Done